test/glific_web/schema/organization_test.exs

```diff
@@ -455,6 +455,7 @@ defmodule GlificWeb.Schema.OrganizationTest do
     assert services["flow_uuid_display"] == false
     assert services["contact_profile_enabled"] == false
     assert services["auto_translation_enabled"] == false
+    assert services["whatsapp_group_enabled"] == false
```

Comment:
how come you have not updated this file
`assets/gql/organizations/get_services.gql`
and still getting false value

Reply:
it is there
![Uploading Screenshot 2024-02-16 at 3.25.05 PM.png…]()